08LessonEight/in-class-project/exercise-06.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Exercise 06</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Recent From Blog</h1>
+        <p>Bacon ipsum dolor amet beef brisket ball tip, alcatra cupim meatball chicken hamburger tenderloin doner porchetta.</p>
+    </header>
+    <main class="clearfix">
+        <article class="box left">
+            <div class="keyboardimage">

```

Comment:
For some reason, this Keyboard.jpg image is broken. Does the src match the file name?